code/datums/outfits/outfit.dm

```diff
@@ -272,11 +272,14 @@ var/list/outfits_decls_by_type_
 
 //Wrapper for creating, so that we can manipulate the items
 /decl/hierarchy/outfit/proc/create_item(var/path, var/location, var/dummy = FALSE)
+	/* As far as I can tell, this is dead code and just causes runtimes.
 	var/datum/thing = path
```

Comment:
no, this is newly added code, and very much active. Fix the bugs with it, don't remove it

Reply:
Thing is, I can't see anywhere where it's used, hence the "bag argument" for dummy. 

Removing it also seemed to not do anything? Weird.

Reply:
it was newly added for future support to prevent full initialisation of items that are only for visuals

if its causing runtimes, fix those

Reply:
It's runtiming because of the new with dummy as a param, but datum doesn't expose a constructor taking in dummy, thus it's causing runtimes inside of BYOND itself. I can extend this to /datum I suppose, but I again am struggling to see what this does, could you point me to a file where it's implemented?

Reply:
This is the sister implementation to the code in loadout_gear.dm. See the code in there
I implemented a var, implements_dummy, which indicates if its safe to pass a dummy parameter to that datum at initialisation.

It is currently being used for rigs and PDAs, to prevent them adding themselves to processing lists when created for mannequin previews.

I may not have finalised that half of the implementation, so fixing it would be good.
If you can think of a more elegant way to approach this problem, i'm all for throwing it out and doing it better